use-cases/demos/classification/tensorflow-keras/Tesnorflow_Keras_Example.ipynb

```diff
@@ -0,0 +1,957 @@
+{
```

Comment:
is this file needed?

Reply:
> is this file needed?

Notebook is for easy and quick visualization and testing. I added it for pytorch so kept it for tensorflow as well.